sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcSourceSuite.scala

```diff
@@ -44,7 +44,7 @@ abstract class OrcSuite extends QueryTest with BeforeAndAfterAll {
     import org.apache.spark.sql.hive.test.TestHive.implicits._
 
     sparkContext
-      .makeRDD(1 to 100)
+      .makeRDD(1 to 10)
```

Comment:
The numbers were increased to 100 to workaround SPARK-8501. Now it's fixed, so revert them back.